libs/application/ui-shell/src/components/FormMultiField.tsx

```diff
@@ -82,11 +84,26 @@ const FormMultiField: FC<
           !IGNORED_HALF_TYPES.includes(field.type) && field?.width === 'half'
         const span = isHalfColumn ? '1/2' : '1/1'
 
+        console.log(application)
+
+        const evaluatedCondition = shouldShowFormItem(
+          field,
+          application.answers,
+          application.externalData,
+          user,
+        )
+
+        const paddingBottom = !evaluatedCondition
+          ? 0
+          : index === children.length - 1
+          ? 0
+          : space
+
```

Comment:
This makes sense, small improvement:

The condition is already being evaluated in the `convertMultiFieldToScreen` reducer function,
it's being passed down to every child field as `isNavigable`

So in theory we could skip this and just do something like

```ts
const paddingBottom = !field.isNavigable ? 0 : index === children.length - 1 ? 0 : space
```